sql/5_0_0-to-5_0_1_upgrade.sql

```diff
@@ -439,3 +470,60 @@ ALTER TABLE `onsite_mail` CHANGE `owner` `owner` varchar(128) DEFAULT NULL;
 #IfNotColumnType openemr_postcalendar_events pc_facility int(11)
 ALTER TABLE `openemr_postcalendar_events` CHANGE `pc_facility` `pc_facility` int(11) NOT NULL DEFAULT '0' COMMENT 'facility id for this event';
 #Endif
+
```

Comment:
When bringing in table in upgrade script, look here for good example:
https://github.com/juggernautsei/openemr/blob/423e230779cbda1d894bb9c140c978ff74b0f00c/sql/5_0_0-to-5_0_1_upgrade.sql#L96